src/engine/plugins/tile.cpp

```diff
@@ -182,6 +182,60 @@ inline void encodePoint(const FixedPoint &pt, protozero::packed_field_uint32 &ge
     geometry.add_element(protozero::encode_zigzag32(dy));
 }
 
+std::vector<FixedLine> coordinatesToTileLine(const std::vector<util::Coordinate> &points,
```

Comment:
I am confused that we wouldn't have logic for this, @danpat how are the other lines generated?

Reply:
The next coordinatesToTileLine function below, but for the single coordinates pair and one FixedLine as a result.